modular_citadel/code/modules/mob/living/silicon/robot/dogborg_equipment.dm

```diff
@@ -283,7 +283,7 @@ SLEEPER CODE IS IN game/objects/items/devices/dogborg_sleeper.dm !
 			L.visible_message("<span class='danger'>[R] has shocked [L] with its tongue!</span>", \
 								"<span class='userdanger'>[R] has shocked you with its tongue!</span>")
 			playsound(loc, 'sound/weapons/Egloves.ogg', 50, 1, -1)
-			R.cell.use(666)
+			R.cell.use(200)
```

Comment:
So borg shocks only take 200 energy now, wtf

Reply:
It was a request by a person, its 20 stam for 666 power when there non-lethal gun can fire have rang and do more for 70~ power